tests/date_time_numeric_functions.rs

```diff
@@ -0,0 +1,816 @@
+use common::compile_and_run;
+
+// Import common functionality into the integration tests
+mod common;
+
+use common::add_std;
+
+#[allow(dead_code)]
+#[derive(Default)]
+#[repr(C)]
+struct MainType {
+    a: i64,
+    b: i64,
+    c: i64,
+    d: i64,
+}
+
+#[test]
+fn add_time() {
+    let src = "
+	PROGRAM main
+	VAR
+		a : TIME;
+		b : TIME;
+		c : LTIME;
+		d : LTIME;
+	END_VAR
+		a := ADD(TIME#5h,TIME#30s);
+		b := ADD_TIME(TIME#10s,TIME#-5s);
+
+		c := ADD(LTIME#-10s,LTIME#-10s);
+		d := ADD_LTIME(LTIME#10s,LTIME#10s);
+	END_PROGRAM";
+    let sources = add_std!(src, "date_time_numeric_functions.st");
+    let mut maintype = MainType::default();
+    let _: i64 = compile_and_run(sources, &mut maintype);
+    assert_eq!(maintype.a, 18030000000000);
```

Comment:
This is very difficult to verify, could you try to create a duration here for the verification? from hours/minutes and so on?